app.py

```diff
@@ -28,12 +38,43 @@ def sign_up_page():
 
 @app.route('/sign_up', methods=['POST'])
 def process_sign_up():
-    if sign_up():
-        return "process completed Eden", 201
-    return "failed to create a new user", 401
+    if process_sign_up():
+        return render_template('signup_completed.html'), 200
+    return render_template('signup_railed.html'), 400
+
+
+@app.route('/login')
+def return_login_page():
+    return render_template('login_form.html')
+
 
+@app.route('/login', methods=['POST'])
+def process_login():
+    if login_verification():
+        return render_template('login_completed.html'), 200
+    return render_template('login_failed.html'), 401
```

Comment:
Same comment. Login failed shouldn't be a new file but just an error inside the login page.
You also shouldn't have "login completed" file. 
Try to think of the minimum amount of HTML files you need here.